test/mitmproxy/tservers.py

```diff
@@ -122,6 +122,7 @@ def get_proxy_config(cls):
             cadir = cls.cadir,
             authenticator = cls.authenticator,
             add_upstream_certs_to_client_chain = cls.add_upstream_certs_to_client_chain,
+            ssl_insecure = True,
```

Comment:
I think this changes the default for all HTTP/2 tests - maybe we can only set this to true in the tests we actually want it to change?